app/bank/domain/transaction_repository.py

```diff
@@ -0,0 +1,18 @@
+from abc import ABC, abstractmethod
+
+from bank.domain.account import Account
+
+
+class TransactionRepository(ABC):
```

Comment:
The name of this abstraction should be 
```suggestion
class AccountRepository(ABC):
```